core/logs/log.go

```diff
@@ -764,9 +764,7 @@ func formatLog(f interface{}, v ...interface{}) string {
 		if len(v) == 0 {
 			return msg
 		}
-		if strings.Contains(msg, "%") && !strings.Contains(msg, "%%") {
-			// format string
-		} else {
+		if !strings.Contains(msg, "%") {
```

Comment:
I think it doesn't resolve the problem.
```
formatLog("%d%%, ", 12, "hello") // expected `12%, hello`
```
does it misses "hello"?

Reply:
The syntax is incorrect. Two placeholders are required for two parameters. You only provide one.
```go
Calling fmt.Sprintf("%d%%", 10,"hello") will not solve this problem.
```

Reply:
@AllenX2018 Sorry, I write a test:
```go
func Test_format(t *testing.T) {
	res := formatLog("%d%%", 10)
	assert.Equal(t, "10%", res)

	res = formatLog("%d", 12)
	assert.Equal(t, "12", res)

	res = formatLog("%%", 12)
	assert.Equal(t, "% 12", 12)
}
```

And it failed.

Reply:
```go
res = formatLog("%%%d", 12)
assert.Equal(t, "%12", 12)
```